### DISCUSSION TITLE
Expulser du jeux lors de la création de taches (multijoueur)

### DISCUSSION THREAD

**Discussion by kevin564601** [Courseplay/Courseplay_FS22#3075] [Q&A]
Bonjours, Nous sommes en multijoueur et lors d'une création de taches je me fait expulser du jeux.
Nous étions chez verygames en serveur de jeux et nous avons changer pour Nitrado et la ca ne fonctionne plus.
Log sans erreur 200mgb en connexion internet avec un ping a 40.
Avez vous des solutions a m'apporter 
Merci d'avance

**Comment by Tensuko** [comment]
@Lactic68 you may help out? I don't know whats the problem.
If it is about disconnection, I can't give a fix for it as we don't know what cause it.

**Reply by kevin564601** [reply]
hello, yes it is a disconnection every time I search has created a task. (translation reverso)

**Comment by Lactic68** [comment]
Bonjour @kevin564601 
Pouvez-vous poster également le log à la fois côté client et aussi côté serveur svp ?
Est-ce que le problème se pose également avec d'autres joueurs ?

@Tensuko 
On dedicated server (hosted by nitrado) when Kevin tries to create a CP task he is kicked out the game.
I have asked him to provide log files both server and client side as well as if he is the only user with this issue.
